docs/using.md

```diff
@@ -15,6 +15,7 @@
         3.  [What is a phase?](#orgb655d69)
         4.  [How do I run a workload?](#org32b8ad3)
     5.  [Outputs](#orgec88ad4)
+        1.  [Analyzing workload output locally](#org2f7c9d4)
```

Comment:
What do you mean by "not putting anything here"? Not sure I follow what you mean. I am going to propose using a human readable `id` rather than an opaque hash. Does that work?